docs/python_support/quickstart.md

```diff
@@ -0,0 +1,222 @@
+        text = bytes(input_data).decode()
+        output_data = text.upper().encode()
+        return [Tensor("OUTPUT", output_data)]
+ ```
+
+Let's create `model.py` file with that code and save it to `workspace/models/python_model`
+
+```bash
+echo '
+from pyovms import Tensor
+
+class OvmsPythonModel:
+
+    def initialize(self, kwargs: dict):
+        return True
+
+    def execute(self, inputs: list):
+        input_data = inputs[0]
+        text = bytes(input_data).decode()
+        output_data = text.upper().encode()
+        return [Tensor("OUTPUT", output_data)]
+' >> models/python_model/model.py
+```
+
+### Step 4: Prepare Graph Configuration File
+
+Python logic execution in OVMS is supported via Mediapipe graphs. That means you'll need to prepare graph definition for your processing flow. In that case, a graph with just one node - Python node - is enough. Let's create appropriate `graph.pbtxt` file in your `workspace/models/python_model` catalog:
+
+```bash
+echo '
+input_stream: "OVMS_PY_TENSOR:text"
+output_stream: "OVMS_PY_TENSOR:uppercase"
+
+node {
+  name: "pythonNode"
+  calculator: "PythonExecutorCalculator"
+  input_side_packet: "PYTHON_NODE_RESOURCES:py"
+  input_stream: "INPUT:text"
+  output_stream: "OUTPUT:uppercase"
+  node_options: {
+    [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+      handler_path: "/models/python_model/model.py"
+    }
+  }
+}' >> models/python_model/graph.pbtxt
+```
+
+Above configuration file creates a graph with a single Python node that uses `PythonExecutorCalculator`, set's inputs and outputs and provides your Python code location in `handler_path`. 
+The `input_side_packet` value is an internal field used by the model server to share resources between graph instances - do not change it. 
+
+### Step 5: Prepare Server Configuration File
+
+Last piece of configuration would be the model server configuration file. 
+Create `config.json` with the following content in `workspace/models`:
+
+```bash
+echo '
+{
+    "model_config_list": [],
+    "mediapipe_config_list": [
+    {
+        "name":"python_model",
+        "graph_path":"/models/python_model/graph.pbtxt"
+    }
+    ]
+}
+' >> models/config.json
+```
+
+### Step 6: Deploy OpenVINO Model Server
+
+Before running the server let's check if you have all the required files. You can do that with a `tree` command:
+```bash
+tree
+.
+├── client.py
+└── models
+    ├── config.json
+    └── python_model
+        ├── graph.pbtxt
+        └── model.py
+```
+
+Now let's run the server:
+```bash
+docker run -it --rm -p 9000:9000 -v $PWD/models:/models openvino/model_server:latest --config_path /models/config.json --port 9000
+```
+### Step 7: Send Requests From The Client
+
+Once you have model server up and running, let's send a text: `"Make this text uppercase."`. 
+
+Simply run your `client.py` from the `workspace` catalog and see the results:
+
+```bash
+python3 client.py
+MAKE THIS TEXT UPPERCASE.
+```
+
```

Comment:
Should we add this page as a link to some other page?

Reply:
I'll link it